Tests/Unit/MSFT_xADDomainController.Tests.ps1

```diff
@@ -23,10 +23,13 @@ $TestEnvironment = Initialize-TestEnvironment `
 try
 {
     #region Pester Test Initialization
-    $correctSiteName = 'PresentSite'
-    $incorrectSiteName = 'IncorrectSite'
-    $correctDomainName = 'present.com'
+    $correctDomainName   = 'present.com'
     $testAdminCredential = [System.Management.Automation.PSCredential]::Empty
+    $correctDatabasePath = 'C:\Windows\NTDS'
+    $correctLogPath      = 'C:\Windows\NTDS'
```

Comment:
>'C: [](start = 27, length = 3)

It is better to  use environmental variable like "$env:SystemDrive\Windoes\NTDS" instead of hard coding the path.